<launch>
  <arg name="traj_order" 		value="3" /><!-- 轨迹阶数 -->
  <!--<arg name="control_points_number"	value="30" />-->
  <arg name="dimension" 		value="2" /><!-- 2维 -->
  <arg name="TrajSampleRate" 		value="50" /><!-- 10, 20, 30 , ... ,-->
<!--  <arg name="beta" 			value="2.0" />-->
  <arg name="max_vel" 			value="1.5" />
  <arg name="max_acc" 			value="2.5" /><!-- 最大速度与加速度 -->

  <arg name="goal_x" 			value="9.0" />
  <arg name="goal_y" 			value="-10.0" />
  <arg name="lambda1" 			value="5.0" />
  <arg name="lambda2" 			value="1.0" />
  <arg name="lambda3" 			value="10.0" />    <!-- 光滑 可行 避障权重 -->
  <arg name="esdf_collision" 		value="1.8" /> <!-- 小于此值则认为轨迹有碰撞 -->
  <arg name="frame" 			value="world" />     <!-- 参考系 -->
  <arg name="map_resolution" 		value="0.1" />
  <arg name="start_x" 			value="-39.95" />
  <arg name="start_y" 			value="39.95" />   <!-- 地图起始点 -->
  <arg name="safe_distance" 		value="6.0" /> <!-- 安全距离 -->
  <arg name="dist_p" 			value="0.5" />       <!-- 均匀b样条每段长度：0.5 -->
  <!--启动规划器，生成轨迹 -->
  <node pkg="bspline_race" name="flag_planner" type="flag_planner" output="screen">	
    <!--<rosparam command="load" file="$(find bspline_opt)/con0fig/initial_final_state.yaml" />-->
    <param name="planning/traj_order" value="$(arg traj_order)"/>
   <!-- <param name="planning/control_points_number" value="$(arg control_points_number)" />-->
    <param name="planning/dimension" value="$(arg dimension)" />
    <param name="planning/TrajSampleRate" value="$(arg TrajSampleRate)" />
    <!--<param name="planning/beta" value="$(arg beta)" />-->
    <param name="planning/max_vel" value="$(arg max_vel)" />
    <param name="planning/max_acc" value="$(arg max_acc)" />
    <param name="planning/goal_x" value="$(arg goal_x)" />	
    <param name="planning/goal_y" value="$(arg goal_y)" />
    <param name="planning/lambda1" value="$(arg lambda1)" />
    <param name="planning/lambda2" value="$(arg lambda2)" />
    <param name="planning/lambda3" value="$(arg lambda3)" />
    <param name="planning/frame" value="$(arg frame)" />
    <param name="planning/map_resolution" value="$(arg map_resolution)" />
    <param name="planning/start_x" value="$(arg start_x)" />
    <param name="planning/start_y" value="$(arg start_y)" />
    <param name="planning/safe_distance" value="$(arg safe_distance)" />
    <param name="planning/esdf_collision" value="$(arg esdf_collision)" />
    <param name="planning/dist_p" value="$(arg dist_p)" />
	
  </node>
  <!--启动rviz，可视化显示 -->
  
</launch>
